src/ui/common/src/components/integrations/dialogs/dialog.tsx

```diff
@@ -482,23 +482,24 @@ export function isConfigComplete(
   service: Service
 ): boolean {
   switch (service) {
-    case 'S3':
-      return isS3ConfigComplete(config as S3Config);
     case 'Athena':
       return isAthenaConfigComplete(config as AthenaConfig);
-    case 'Kubernetes':
-      return isK8sConfigComplete(config as KubernetesConfig);
     case 'Conda':
       return true;
     case 'Databricks':
       return isDatabricksConfigComplete(config as DatabricksConfig);
     case 'Email':
       return isEmailConfigComplete(config as EmailConfig);
+    case 'Kubernetes':
```

Comment:
Just a little alphabetical sorting here

Reply:
Thanks for doing this. I wonder if there is a linter rule that we can apply here, I'll see if I can find something like that.

Perhaps something like this could benefit us in the future:
https://palantir.github.io/tslint/rules/member-ordering/